src/Rules/Rule.cs

```diff
@@ -13,12 +13,16 @@ interface IRule
         RuleInfo Info { get; }
         Condition Condition { get; }
         bool PassesTest(IA11yElement element);
+        bool IncludeInResults(IA11yElement element);
+        // True if this rule failing should cause the element to be excluded from the rest of the scan.
+        bool Exclusionary { get; }
```

Comment:
How would you feel about naming this field `IsExclusionRule` to align with how it's consumed?